controllers/component_build_controller_pac.go

```diff
@@ -1119,7 +1133,8 @@ func generatePaCPipelineRunForComponent(
 	additionalPipelineParams []tektonapi.Param,
 	onPull bool,
 	pacTargetBranch string,
-	gitClient gp.GitProviderClient) (*tektonapi.PipelineRun, error) {
+	gitClient gp.GitProviderClient,
+	pipelineLabelUsed bool) (*tektonapi.PipelineRun, error) {
```

Comment:
I see no point in this parameter. The value can be easily obtained from the component.